zkevm-circuits/src/evm_circuit.rs

```diff
@@ -1296,4 +1302,87 @@ mod test {
                 .assign(&mut layouter, &self.execution_steps)
         }
     }
+
+    // To run this benchmark, comment the `ignore` flag and run the following
+    // command:
+    // `RUSTFLAGS='-C target-cpu=native' cargo test --profile bench
+    // bench_evm_circuit_prover -- --nocapture`
+    #[ignore]
+    #[test]
+    fn bench_evm_circuit_prover() {
+        const DEGREE: u32 = 22;
```

Comment:
As @icemelon said. We can customize the bench more if we go for the env_var + lazy_static approach to generate a const DEGREE value from the `BENCH_DEGREE` env var for example. 

Do you see that as a better option? @ed255 @han0110 ? Since I liked it TBH

Reply:
Not familiar with benchmark but using env to adjust `DEGREE` sounds good to me.